docs/admin_guide.md

```diff
@@ -158,6 +158,155 @@ hoodie:trips->stats wa
+                 Help:                   Parallelism for hoodie compaction
+                 Mandatory:              true
+                 Default if specified:   '__NULL__'
+                 Default if unspecified: '__NULL__'
+              
+               Keyword:                  schemaFilePath
+                 Help:                   Path for Avro schema file
+                 Mandatory:              true
+                 Default if specified:   '__NULL__'
+                 Default if unspecified: '__NULL__'
+              
+               Keyword:                  sparkMemory
+                 Help:                   Spark executor memory
+                 Mandatory:              true
+                 Default if specified:   '__NULL__'
+                 Default if unspecified: '__NULL__'
+              
+               Keyword:                  retry
+                 Help:                   Number of retries
+                 Mandatory:              true
+                 Default if specified:   '__NULL__'
+                 Default if unspecified: '__NULL__'
+              
+              * compaction schedule - Schedule Compaction
+```
+
+```
+hoodie:trips->help compaction run
+
+Keyword:                   compaction run
+Description:               Run Compaction for given instant time
+ Keyword:                  tableName
+   Help:                   Table name
+   Mandatory:              true
+   Default if specified:   '__NULL__'
+   Default if unspecified: '__NULL__'
+
+ Keyword:                  rowKeyField
+   Help:                   Row key field name
+   Mandatory:              true
+   Default if specified:   '__NULL__'
+   Default if unspecified: '__NULL__'
+
+ Keyword:                  parallelism
+   Help:                   Parallelism for hoodie compaction
+   Mandatory:              true
+   Default if specified:   '__NULL__'
+   Default if unspecified: '__NULL__'
+
+ Keyword:                  schemaFilePath
+   Help:                   Path for Avro schema file
+   Mandatory:              true
+   Default if specified:   '__NULL__'
+   Default if unspecified: '__NULL__'
+
+ Keyword:                  sparkMemory
+   Help:                   Spark executor memory
+   Mandatory:              true
+   Default if specified:   '__NULL__'
+   Default if unspecified: '__NULL__'
+
+ Keyword:                  retry
+   Help:                   Number of retries
+   Mandatory:              true
+   Default if specified:   '__NULL__'
+   Default if unspecified: '__NULL__'
+
+ Keyword:                  compactionInstant
+   Help:                   Base path for the target hoodie dataset
+   Mandatory:              true
+   Default if specified:   '__NULL__'
+   Default if unspecified: '__NULL__'
+
+* compaction run - Run Compaction for given instant time
+```
+
+##### Up-Coming CLI for Compaction
+
+In the next release, new useful CLI to revert/repair compaction schedules will be added. Here is a preview of them:
```

Comment:
Change to : 

In the next release, more useful CLI to revert/repair compaction schedules will be added.